examples/AccountManagement/GetAccountHierarchy.cs

```diff
@@ -0,0 +1,218 @@
+        {
+             // googleAdsServiceClient = googleAdsClient.GetService(Services.V3.GoogleAdsService);
+            
+            // List of Customer IDs to handle.
+            ArrayList seedCustomerIds = new ArrayList();
+            
+            // If a Manager ID was provided above, it will be the only ID in the list.
+            // Otherwise, we will issue a request for all customers accessible by this authenticated Google account.
+            if(customerId.HasValue)
+            {
+                seedCustomerIds.Add(customerId);
+            }
+            else
+            {
+                Console.WriteLine("No manager customer ID is specified. The example will print the hierarchies of " + 
+                                  "all accessible customer IDs:");
+                CustomerServiceClient customerServiceClient = googleAdsClient.GetService(Services.V3.CustomerService);
+                string[] accessibleCustomers =  customerServiceClient.ListAccessibleCustomers();
+
+                foreach (string customerResourceName in accessibleCustomers)
+                {
+                    Customer customer = customerServiceClient.GetCustomer(customerResourceName);
+                    Console.WriteLine(customer.Id);
+                    seedCustomerIds.Add(customer.Id);
+                }
+                Console.WriteLine();
+            }
+            
+            // Creates a query that retrieves all child accounts of the manager specified in search calls below.
+            const string query = "SELECT customer_client.client_customer, customer_client.level,"
+                                 + " customer_client.manager, customer_client.descriptive_name,"
+                                 + " customer_client.currency_code, customer_client.time_zone,"
+                                 + " customer_client.id FROM customer_client WHERE customer_client.level <= 1";
+
+            foreach (long seedCustomerId in seedCustomerIds)
+            {
+                GoogleAdsServiceClient googleAdsServiceClient = googleAdsClient.GetService(Services.V3.GoogleAdsService);
+                
+                // Performs a breadth-first search to build an associative array mapping
+                // managers to their child accounts (customerIdsToChildAccounts).
+                Queue<long> unprocessedCustomerIds = new Queue<long>();
+                unprocessedCustomerIds.Enqueue(seedCustomerId);
+                Dictionary<long, ArrayList> customerIdsToChildAccounts =
+                    new Dictionary<long, ArrayList>();
+                CustomerClient rootCustomerClient = null;
+
+                while (unprocessedCustomerIds.Count > 0)
+                {
+                    customerId = unprocessedCustomerIds.Dequeue();
+                    PagedEnumerable<SearchGoogleAdsResponse, GoogleAdsRow> response = 
+                        googleAdsServiceClient.Search(customerId.ToString(), query, pageSize: PAGE_SIZE);
```

Comment:
For readability, we usually put all the arguments on their own line. It may take more space vertically, but it's often easier for people to spot things. Plus by putting more vertically, we can do includes into our documentation a lot easier without pulling in extraneous information.

Reply:
Fixed.